airbyte-integrations/connectors/source-faker/unit_tests/unit_test.py

```diff
@@ -251,30 +262,16 @@ def test_read_with_seed():
     config = {"count": 1, "seed": 100, "parallelism": 1}
     catalog = ConfiguredAirbyteCatalog(
         streams=[
-            {
-                "stream": {"name": "users", "json_schema": {}, "supported_sync_modes": ["incremental"]},
-                "sync_mode": "incremental",
-                "destination_sync_mode": "overwrite",
-            }
+            ConfiguredAirbyteStream(
+                stream=AirbyteStream(name="users", json_schema={}, supported_sync_modes=["incremental"]),
+                sync_mode="incremental",
+                destination_sync_mode="overwrite",
+            )
         ]
     )
     state = {}
     iterator = source.read(logger, config, catalog, state)
 
     records = [row for row in iterator if row.type is Type.RECORD]
-    assert records[0].record.data["occupation"] == "Sheriff Principal"
-    assert records[0].record.data["email"] == "alleged2069+1@example.com"
-
-
-def test_ensure_no_purchases_without_users():
```

Comment:
@evantahler what is the intent for this test?

On master, the error raised is
```
E           pydantic_core._pydantic_core.ValidationError: 1 validation error for ConfiguredAirbyteCatalog
E           streams.0.stream.supported_sync_modes
E             Field required [type=missing, input_value={'name': 'purchases', 'json_schema': {}}, input_type=dict]
E               For further information visit https://errors.pydantic.dev/2.10/v/missing
```
which has nothing to do with purchases not being valid without users. I didn't see any code checking that either for what it's worth

Reply:
I think this was testing that you can't sync the purchase stream without also syncing the users stream.  But... that's a silly requirement